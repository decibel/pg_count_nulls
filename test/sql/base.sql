\set ECHO 0
BEGIN;
\i sql/pg_numnulls.sql
\set ECHO all

-- You should write your tests

SELECT pg_numnulls('foo', 'bar');

SELECT 'foo' #? 'bar' AS arrowop;

CREATE TABLE ab (
    a_field pg_numnulls
);

INSERT INTO ab VALUES('foo' #? 'bar');
SELECT (a_field).a, (a_field).b FROM ab;

SELECT (pg_numnulls('foo', 'bar')).a;
SELECT (pg_numnulls('foo', 'bar')).b;

SELECT ('foo' #? 'bar').a;
SELECT ('foo' #? 'bar').b;

ROLLBACK;
