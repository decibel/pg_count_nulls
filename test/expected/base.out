\set ECHO 0
unrecognized value "0" for "ECHO"; assuming "none"
psql:sql/pg_numnulls.sql:8: ERROR:  type anyelement[] does not exist
-- You should write your tests
SELECT pg_numnulls('foo', 'bar');
ERROR:  current transaction is aborted, commands ignored until end of transaction block
SELECT 'foo' #? 'bar' AS arrowop;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
CREATE TABLE ab (
    a_field pg_numnulls
);
ERROR:  current transaction is aborted, commands ignored until end of transaction block
INSERT INTO ab VALUES('foo' #? 'bar');
ERROR:  current transaction is aborted, commands ignored until end of transaction block
SELECT (a_field).a, (a_field).b FROM ab;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
SELECT (pg_numnulls('foo', 'bar')).a;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
SELECT (pg_numnulls('foo', 'bar')).b;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
SELECT ('foo' #? 'bar').a;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
SELECT ('foo' #? 'bar').b;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
ROLLBACK;
